src/docs/015_tasks/03_task_copy_themes.adoc

```diff
@@ -2,21 +2,19 @@
 include::_config.adoc[]
 
 [[task-copyThemes]]
-= copyThemes

```

Comment:
I've switched from section level 0 to level 1 here because IntelliJ inspections were complaining about it being only allowed in documents with type book. But after comparing it with AsciiDoctor documentation source where level 0 is the highest level inside a single page I am no longer sure about that change.

@rdmueller what is the expected section level here and how does it translate to the microsite generated by JBake? Or is there already some documentation regarding the appropriate section levels?

Reply:
I always use level 1 (`==`) in my documentation files because they are only chapters. When I render for example a PDF, those chapters will be assembled to a full document which gets a level 0 (`=`) headline. Level 0 is only allowed once in a document.